### DISCUSSION TITLE
Issue with illegal characters and word in page creation.

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3578] [Error / Bug Report]
**Describe the bug**
Space and word used to create path.

**To Reproduce**
1)
Create new page.
Add space between some word, such has / Google/G Suite.

This will cause the creation to fail with an error message saying you have illegal characters.

2)
Create new page with the top folder to be / KB/Path/To/Folder.

It will crash the path and you won't be able to do much with it. You will see some odd changes on the Creation page.

**Expected behavior**

**Screenshots**
1)
![image](https://user-images.githubusercontent.com/67117859/86042753-a0054180-ba15-11ea-836d-4e69c514f9c9.png)
![image](https://user-images.githubusercontent.com/67117859/86042809-b3181180-ba15-11ea-9c2b-49938bdbdd01.png)

2)
![image](https://user-images.githubusercontent.com/67117859/86042929-ea86be00-ba15-11ea-893d-14e72c8b3ef0.png)
![image](https://user-images.githubusercontent.com/67117859/86043001-08542300-ba16-11ea-9a3d-18346e9a12a3.png)

**Host Info (please complete the following information):**
 - OS: Docker
 - Wiki.js version: [e.g. 2.0.1]
 - Database engine: postgres 9.7

**Comment by ghost** [comment]
Issue 1:
I believe the docs has already stated that spaces are considered an illegal character. Or did i misunderstood your question?
![image](https://user-images.githubusercontent.com/9043555/86546430-76916c00-bf67-11ea-934d-c4b91280413b.png)

Issue 2:
As for this issue, was the Locale "KB" a valid locale?

**Comment by sebi-** [comment]
We have the same usability issue as above step 1. You can also duplicate a page for example, and enter spaces or other illegal characters in the path, and click "Select" without issue. You can then edit the page, but until you click on "create", you never see the error message about the illegal characters. This should already be caught in the path selection dialog, by (i) explaining the restriction and (ii) not letting you click "Select" in the path selection dialog without an error message if illegal characters are entered.
